projects/plugins/jetpack/json-endpoints/class.wpcom-json-api-get-site-endpoint.php

```diff
@@ -72,6 +72,7 @@ class WPCOM_JSON_API_GET_Site_Endpoint extends WPCOM_JSON_API_Endpoint {
 		'quota'                       => '(array) An array describing how much space a user has left for uploads',
 		'launch_status'               => '(string) A string describing the launch status of a site',
 		'site_migration'              => '(array) Data about any migration into the site.',
+		'is_favorite'                 => '(bool) If the user marked a site as favorite.',
```

Comment:
I think we'll want to call this `is_user_favorite` to denote that it's a value of the user, not a site-based property.

Reply:
Thanks @danielbachhuber  - we updated it to `is_user_favorite`